pkg/config/config.go

```diff
@@ -1174,6 +1174,7 @@ func InitConfig(config Config) {
 	// Datadog security agent (compliance)
 	config.BindEnvAndSetDefault("compliance_config.enabled", false)
 	config.BindEnvAndSetDefault("compliance_config.xccdf.enabled", false)
```

Comment:
could you add a ` // deprecated` comment here ?

Reply:
Done.